Week4/app/src/main/java/com/isabelledionisius/mymovieapp/DetailActivity.kt

```diff
@@ -0,0 +1,36 @@
+package com.isabelledionisius.mymovieapp
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.EditText
+import android.widget.ImageView
+import android.widget.TextView
+import kotlin.properties.Delegates
+
+class DetailActivity : AppCompatActivity() {
+    /**
+     * properties to be displayed in the detail view
+     */
+    lateinit var movies:Movie
```

Comment:
I think I mentioned this before, but try using Kotlin Syntetix (Kotlin Android Extensions), or ViewBinding. It's much easier and better than lateinit vars!